10-inference.Rmd

```diff
@@ -0,0 +1,789 @@
+          y = unit(0.04, "npc"),gp = gpar(fontsize=15))
+grid.text(".", x = unit(0.45, "npc"), 
+          y = unit(0.06, "npc"),gp = gpar(fontsize=15))
+grid.text(".", x = unit(0.45, "npc"), 
+          y = unit(0.025, "npc"),gp = gpar(fontsize=15))
+
+grid.text(".", x = unit(0.7, "npc"), 
+          y = unit(0.14, "npc"),gp = gpar(fontsize=15))
+grid.text(".", x = unit(0.7, "npc"), 
+          y = unit(0.16, "npc"),gp = gpar(fontsize=15))
+grid.text(".", x = unit(0.7, "npc"), 
+          y = unit(0.18, "npc"),gp = gpar(fontsize=15))
+
+```
+
+### Using the bootstrap to calculate a plausible range  
+
+Now that we have constructed our bootstrap distribution let's use it to create
+an approximate bootstrap confidence interval, a range of plausible values for the population mean. We will construct a 95\% percentile bootstrap confidence interval and find the range of values that cover the middle 95\% of the bootstrap distribution. Note that there's nothing particularly special about 95\%, we could have used other confidence levels, such as 90\% or 99\%. The confidence level you choose will depend on your application, which is beyond the scope of this textbook.
+
+To calculate our 95\% percentile bootstrap confidence interval, we will do the following:
+
+1. Arrange the observations in the bootstrap distribution in ascending order 
+2. Find the value such that 2.5\% of observations fall below it (the 2.5\% percentile). Use that value as the lower bound of the interval 
+3. Find the value such that 97.5\% of observations fall below it (the 97.5\% percentile). Use that value as the upper bound of the interval
+
+To do this in R, we can use the `quantile()` function:
+
+```{r 11-bootstrapping8, echo = T, message = FALSE, warning = FALSE}
+bounds <- boot1500_means %>% 
+    select(mean) %>% 
+    pull() %>% 
+    quantile(c(0.025, 0.975))
+bounds
+```
+
+Our interval, \$`r round(bounds[1],2) ` to \$`r round(bounds[2],2)`, captures the middle 95\% of the sample mean prices in the bootstrap distribution. We can visualize the interval on our distribution in the picture below. 
+
+```{r 11-bootstrapping9, echo = F, message = FALSE, warning = FALSE, fig.cap = "Distribution of the bootstrap sample means with percentile lower and upper bounds"}
+boot_est_dist + 
+  geom_vline(xintercept = bounds, col = "#E69F00", size = 2, linetype = 2) + 
+  annotate("text", x = bounds[1], max_count(boot_est_dist), hjust = 0.5, vjust =2,
+           label = paste("2.5th percentile =", round(bounds[1], 2))) + 
+    annotate("text", x = bounds[2], max_count(boot_est_dist), hjust = 0.5, vjust =2,
+           label = paste("97.5th percentile =", round(bounds[2], 2)))
+```
+
+Here we notice that our interval does indeed contain the true
+population mean value, \$`r round(mean(airbnb$price),2)`\! However, in
+practice, we would not know whether our interval captured the true population
+parameter or not because we usually only have a single sample, not the entire
+population.
+
```

Comment:
Maybe add however, this is the best we can do when we only have one sample!

Let's add how to report our point estimate and confidence intervals, and revisit what a confidence intervals means. Maybe loop back to the question we asked at the beginning of this section.

Finally, perhaps end on this is only the beginning of the journey into statistical inference, and we can extend the concepts learned here to do much more than report point estimates and confidence intervals, such as hypothesis testing for differences between populations, associations between variables, etc, but what we presented here is the foundation for this... something to wrap this up nicely with a 🎁